frontend/src/components/PurposeConfiguration.vue

```diff
@@ -62,12 +62,17 @@ export default {
   },
   computed: {
     ...mapGetters([
-      'infrastructureSecretsByCloudProfileName'
+      'infrastructureSecretsByCloudProfileName',
+      'canGetSecrets'
     ]),
     valid () {
       return this.purposeValid
     },
     secret () {
+      if (!this.canGetSecrets) {
+        return undefined
```

Comment:
how does this influence the purpose options? I think, that if no secret is set you'll get the whole list, even if the secret is e.g. a trial secret. Is this what we want here?

Reply:
✅